docs/user-guide/mapstore-toolbars.md

```diff
@@ -169,6 +169,20 @@ In particular, with these options it is possible to:
 
 <img src="../img/menu-bar/settings-panel.jpg" class="ms-docimage" style="max-width:400px;"/>
 
+!!!Note
+    When the [3D navigation](side-bar.md#3d-navigation) is enabled, opening the **Settings** panel, the editor is allowed to configure some options related to the *Cesium viewer*.
+    <img src="../img/menu-bar/cesium-settings.jpg" class="ms-docimage" style="max-width:500px;"/>
+
+    In particular, from the *Map Settings* it is possible to:
+
+    * Enable the **Show sky atmosphere** to see the atmosphere around the globe
+
+    * Enable the **Show ground atmosphere** to view the ground atmosphere on the globe when viewed from a distance
```

Comment:
* Enable the **Show ground atmosphere** to view the ground atmosphere on the globe when the camera is far away